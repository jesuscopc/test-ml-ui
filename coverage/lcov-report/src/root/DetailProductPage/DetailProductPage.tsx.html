
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for src/root/DetailProductPage/DetailProductPage.tsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../index.html">All files</a> / <a href="index.html">src/root/DetailProductPage</a> DetailProductPage.tsx</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>0/7</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>0/7</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a></td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js"><span class="cstat-no" title="statement not covered" >import React from 'react';</span>
<span class="cstat-no" title="statement not covered" >import WithHeader from 'hocs/WithHeader/WithHeader';</span>
<span class="cstat-no" title="statement not covered" >import Ipod from 'assets/images/ipod.jpg';</span>
<span class="cstat-no" title="statement not covered" >import './index.scss';</span>
&nbsp;
const DetailProductPage = <span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >() =</span>&gt; {</span>
<span class="cstat-no" title="statement not covered" >  return (</span>
    &lt;div className="container detail"&gt;
      &lt;section className="detailProduct"&gt;
        &lt;img src={Ipod} alt="ipod touch 5g" width="200" height="200" tabIndex={0} /&gt;
        &lt;h2&gt;Descipcion del producto&lt;/h2&gt;
        &lt;p title="descripion del producto" tabIndex={0}&gt;
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque deleniti laboriosam enim
          dolorem natus eaque quae quod voluptatem necessitatibus, saepe vero praesentium quam
          temporibus nihil accusantium sit! Repellendus, molestias sunt!
        &lt;/p&gt;
      &lt;/section&gt;
      &lt;section className="buyCard"&gt;
        &lt;p className="buyCard__status" title="items vendidos" tabIndex={0}&gt;
          220 vendidos
        &lt;/p&gt;
        &lt;p className="buyCard__title" title="producto" tabIndex={0}&gt;
          Deco reverse sombrero oxxo
        &lt;/p&gt;
        &lt;p className="buyCard__price" title="precio" tabIndex={0}&gt;
          $ 1980&lt;small&gt;00&lt;/small&gt;
        &lt;/p&gt;
        &lt;button type="button"&gt;Comprar&lt;/button&gt;
      &lt;/section&gt;
    &lt;/div&gt;
  );
};
&nbsp;
<span class="cstat-no" title="statement not covered" >export default WithHeader(DetailProductPage);</span>
&nbsp;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Sun Jul 04 2021 14:09:25 GMT-0500 (Central Daylight Time)
            </div>
        </div>
        <script src="../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../sorter.js"></script>
        <script src="../../../block-navigation.js"></script>
    </body>
</html>
    