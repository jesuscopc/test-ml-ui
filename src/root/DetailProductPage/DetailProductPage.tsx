import React from 'react';
import WithHeader from 'hocs/WithHeader/WithHeader';
import Ipod from 'assets/images/ipod.jpg';
import './index.scss';

const DetailProductPage = () => {
  return (
    <div className="container detail">
      <section className="detailProduct">
        <img src={Ipod} alt="ipod touch 5g" width="200" height="200" tabIndex={0} />
        <h2>Descipcion del producto</h2>
        <p title="descripion del producto" tabIndex={0}>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque deleniti laboriosam enim
          dolorem natus eaque quae quod voluptatem necessitatibus, saepe vero praesentium quam
          temporibus nihil accusantium sit! Repellendus, molestias sunt!
        </p>
      </section>
      <section className="buyCard">
        <p className="buyCard__status" title="items vendidos" tabIndex={0}>
          220 vendidos
        </p>
        <p className="buyCard__title" title="producto" tabIndex={0}>
          Deco reverse sombrero oxxo
        </p>
        <p className="buyCard__price" title="precio" tabIndex={0}>
          $ 1980<small>00</small>
        </p>
        <button type="button">Comprar</button>
      </section>
    </div>
  );
};

export default WithHeader(DetailProductPage);
