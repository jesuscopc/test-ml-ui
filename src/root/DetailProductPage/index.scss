$minColWidth: 5rem;
.detail {
  margin-top: 35px;
  padding-top: 30px;
  background-color: #fff;
}
.detailProduct {
  grid-column-start: 3;
  grid-column-end: 9;
  & img {
    width: 100%;
    height: auto;
  }
}

.buyCard {
  grid-column-start: 9;
  grid-column-end: 12;
  max-width: 200px;
  &__status {
    font-size: 14px;
    margin-bottom: 16px;
  }
  &__title {
    font-size: 24px;
    font-weight: bold;
    margin-top: 0px;
    margin-bottom: 32px;
  }
  &__price {
    font-size: 40px;
    margin-top: 0px;
    margin-bottom: 32px;
  }
  & button {
    padding: 10px;
    background-color: #3483fa;
    color: white;
    border: none;
    width: 100%;
    font-size: 20px;
    border-radius: 5px;
  }
}

@media (max-width: 768px) {
  .detailProduct {
    grid-column-start: 2;
    grid-column-end: 12;
  }
  .buyCard {
    grid-column-start: 2;
    grid-column-end: 12;
    min-width: 100%;
  }
}
