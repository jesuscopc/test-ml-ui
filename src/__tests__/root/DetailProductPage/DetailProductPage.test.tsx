import React from 'react';
import { mount } from 'enzyme';
import DetailProductPage from 'root/DetailProductPage/DetailProductPage';

describe('Suit test DetailProductPage Page', () => {
  let wrapper = mount(<DetailProductPage />);
  beforeEach(() => {
    wrapper = mount(<DetailProductPage />);
  });

  test('should be render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
