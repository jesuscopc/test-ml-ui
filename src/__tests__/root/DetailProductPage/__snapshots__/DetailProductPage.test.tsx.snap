// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suit test DetailProductPage Page should be render correctly 1`] = `
<Component>
  <div
    className="header"
  >
    <div
      className="container header__navBar"
    >
      <img
        alt="pagina principal"
        className="header__logo"
        src={Object {}}
        tabIndex={0}
      />
      <div
        className="header__searchBox"
      >
        <SearchBox>
          <form
            className="searchBox"
            onSubmit={[Function]}
          >
            <input
              name="search"
              onChange={[Function]}
              placeholder="Nunca dejes de buscar"
              type="search"
              value=""
            />
            <button
              type="submit"
            >
              <img
                alt="buscar"
                src={Object {}}
              />
            </button>
          </form>
        </SearchBox>
      </div>
    </div>
  </div>
  <DetailProductPage>
    <div
      className="container detail"
    >
      <section
        className="detailProduct"
      >
        <img
          alt="ipod touch 5g"
          height="200"
          src={Object {}}
          tabIndex={0}
          width="200"
        />
        <h2>
          Descipcion del producto
        </h2>
        <p
          tabIndex={0}
          title="descripion del producto"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque deleniti laboriosam enim dolorem natus eaque quae quod voluptatem necessitatibus, saepe vero praesentium quam temporibus nihil accusantium sit! Repellendus, molestias sunt!
        </p>
      </section>
      <section
        className="buyCard"
      >
        <p
          className="buyCard__status"
          tabIndex={0}
          title="items vendidos"
        >
          220 vendidos
        </p>
        <p
          className="buyCard__title"
          tabIndex={0}
          title="producto"
        >
          Deco reverse sombrero oxxo
        </p>
        <p
          className="buyCard__price"
          tabIndex={0}
          title="precio"
        >
          $ 1980
          <small>
            00
          </small>
        </p>
        <button
          type="button"
        >
          Comprar
        </button>
      </section>
    </div>
  </DetailProductPage>
</Component>
`;
